lib/ansible/modules/cloud/vmware/vmware_guest_powerstate.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c), Abhijeet Kasurde <akasurde@redhat.com>
```

Comment:
So the syntax for copyright statements is:
```python
# Copyright: (c) 2017, Abhijeet Kasurde <akasurde@redhat.com>
```

Reply:
Done.